src/frontends/tensorflow_common/src/utils.cpp

```diff
@@ -414,6 +431,142 @@ Output<Node> compute_broadcast_args(const Output<Node>& shape1, const Output<Nod
     auto broadcasted_shape = make_shared<v1::Maximum>(padded_s0, padded_s1);
     return broadcasted_shape->output(0);
 }
+
+shared_ptr<tuple<shared_ptr<Node>, shared_ptr<Node>, shared_ptr<Node>>> rgb_to_hsv(
+    const shared_ptr<Node>& images,
+    element::Type type) {
```

Comment:
can you please pass `images` to this function and use `create_same_type_const` or `create_same_type_const_scalar` from utils  instead? That is needed because `type` can be dynamic and not known at this stage of conversion.

Reply:
the same comment for `hsv_to_rgb`